### DISCUSSION TITLE
setupShowOperation & html fields

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#451] [Q&A (Help)]
How can i render a field type wysiwyg in setupShowOperation?

**Comment by tabacitu** [comment]
Hey Mauro,

Show operation doesn't have fields, it has columns. 

To add fields to the Show operation you have to create that logic yourself. If I were you, I'd [create a custom widget](https://backpackforlaravel.com/docs/5.x/base-widgets#creating-a-custom-widget-type-1) that shows a form, and in that form show a QYSIWYG editor. Take note that you have to take care of EVERYTHING in that custom functionality - showing the form, submitting the form, saving the form. It's custom, so it's completely in your control. But it's also your responsibility to code everything.

Hope it helps.
Cheers!
